ckanext/subscribe/templates/package/snippets/info.html

```diff
@@ -23,6 +23,11 @@
       <form method='post' action="{{ h.url_for('/subscribe/signup') }}" id="subscribe-form" enctype="multipart/form-data" class="form-inline">
         <!-- (Bootstrap 3) <div class="form-group input-group-sm"> -->
           <input id="subscribe-email" type="email" name="email" class="form-control input-small" value="" placeholder="" />
+          {% if apply_recaptcha %}
+              <!-- reCAPTCHA  -->
+              <script src="https://www.google.com/recaptcha/api.js" async defer></script>
+              <div class="g-recaptcha" data-sitekey={{ subscription.recaptcha_publickey }} style="transform:scale(0.85);transform-origin:0"></div>
```

Comment:
I'm also not sure where we get `subscription.recaptcha_publickey` from here, am I missing something?

Unless I'm misunderstanding, I think it would be better to add the two new helpers in your other pull request (`get_recaptcha_publickey` and `apply_recaptcha`) into this repository, and then call them in the template here to get those values. We can also use the same helpers in ckanext-switzerland without redefining them there.

You will have to make the `SubscribePlugin` implement `ITemplateHelpers` to be able to do this.